plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImagePage.java

```diff
@@ -231,11 +231,18 @@ private void createProjectControl(Composite parent, DataBindingContext dbc) {
 			.align(SWT.FILL, SWT.CENTER).grab(true, false)
 			.applyTo(cmboProject.getControl());
 		
+		final OpenShiftExplorerLabelProvider labelProvider = new OpenShiftExplorerLabelProvider();
 		cmboProject.setContentProvider(new ObservableListContentProvider());
-		cmboProject.setLabelProvider(new OpenShiftExplorerLabelProvider());
+		cmboProject.setLabelProvider(labelProvider);
 		cmboProject.setInput(
 				BeanProperties.list(IDeployImagePageModel.PROPERTY_PROJECTS).observe(model));
-		
+		cmboProject.setSorter(new ViewerSorter() {
+			@Override
+			public int compare(Viewer viewer, Object e1, Object e2) {
```

Comment:
It should, but maybe we should consider removing it all together.  I'm not sure how the current ording works in the explorer window.  I assume it returns in order of what was returned by the call.  I dont know whay it wouldnt do the same here.